wire-ios-sync-engine/Source/SessionManager/SessionFactories.swift

```diff
@@ -17,10 +17,36 @@
 //
 
 import avs
+import WireAPI
 import WireDataModel
 
+
+final class InMemoryAuthenticationStorage: AuthenticationStorage {
+
+    private var accessToken: WireAPI.AccessToken?
+    private var cookieData: Data?
+
+    func storeAccessToken(_ accessToken: WireAPI.AccessToken) {
```

Comment:
Out of interest, are these custom setters / getters to implement `AuthenticationStorage`?

Reply:
They were a temporary implementation of `AuthenticationStorage`  so that we could check that the app builds and run the tests while John was finalising it.